### PR TITLE
chore: remove ursa-tracker

### PR DESCRIPTION
**Pull request by ozwaldorf**
## Why

Removes `ursa-tracker` crate and all usages. No longer needed since we've moved to a more decentralized Kademlia based solution instead. 

## What

- delete `crates/ursa-tracker`
- remove tracker from configs
- remove tracker registration in ursa main

## Checklist

- [x] ~~I have made corresponding changes to the tests~~
- [x] ~~I have made corresponding changes to the documentation~~
- [x] I have run the app using my feature and ensured that no functionality is broken

### CODE REVIEW

**Review by b0xtch — APPROVED** [review]
Remove:
https://github.com/fleek-network/ursa/blob/a56c50e6b3993837a38e96cbb8e6f2c8fb5a3d46/.gitignore#L21

Remove:
https://github.com/fleek-network/ursa/blob/chore/remove-tracker/docker/global-metrics/data/nginx/app.conf

Remove (anything related to tracker):
https://github.com/fleek-network/ursa/tree/chore/remove-tracker/infra/tf

**Comment by ozwaldorf** [comment]
> Remove: https://github.com/fleek-network/ursa/blob/chore/remove-tracker/docker/global-metrics/data/nginx/app.conf
Still required for https://dashboard.ursa.earth

File: infra/README.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 Please make sure to update tests as appropriate.
 
 ## License
-[MIT](https://choosealicense.com/licenses/mit/)
```
Comment (b0xtch) [inline infra/README.md:33]: what was this change?

**Comment by b0xtch** [comment]
> > Remove: https://github.com/fleek-network/ursa/blob/chore/remove-tracker/docker/global-metrics/data/nginx/app.conf
> 
> This is still required for https://dashboard.ursa.earth, cleanup has already been done there

was it, or is it now after the latest changes?

**Comment by ozwaldorf** [comment]
> > > Remove: https://github.com/fleek-network/ursa/blob/chore/remove-tracker/docker/global-metrics/data/nginx/app.conf
> > 
> > 
> > This is still required for https://dashboard.ursa.earth, cleanup has already been done there
> 
> was it, or is it now after the latest changes?

`docker/global-metrics/*` was refactored in #406, app.conf was reconfigured to only proxy the grafana instance over http and https

**Review by b0xtch — APPROVED** [review]

**Pull request merged**